modules/spo-recording-profiles.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-recording-profiles_{context}"]
+= Recording profiles from workloads
+
+The Security Profiles Operator can record {type} profiles with `ProfileRecording` objects.
+
+When using the log enricher for recording {type} profiles, ensure that the log enricher feature is enabled. See _Additional resources_ for more information.
+
+[NOTE]
+====
+A container with `privileged: true` security context restraints prevents log-based recording. Privileged containers are not subject to {type} policies and log-based recording makes use of a special {type} profile to record events.
+====
+
+.Procedure
+
+. Create a `ProfileRecording` containing a `recorder: logs` variable:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: ProfileRecording
+metadata:
+  name: test-recording
+spec:
+  kind: {kind}
+  recorder: logs
+  podSelector:
+    matchLabels:
+      app: my-app
+----
+
+. Create a workload to record:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: my-pod
+  labels:
+    app: my-app
+spec:
+  containers:
+    - name: nginx
+      image: quay.io/security-profiles-operator/test-nginx:1.19.1
+    - name: redis
+      image: quay.io/security-profiles-operator/redis:6.2.1
+----
+
+. Confirm the pod is in a `Running` state by entering the following command:
++
+[source,terminal]
+----
+$ oc get pods
```

Comment:
Could add the `-n security-profiles-operator` flag here to be consistent with other examples.